enzyme/Enzyme/EnzymeLogic.cpp

```diff
@@ -3645,6 +3664,133 @@ Function *EnzymeLogic::CreateForwardDiff(
     }
   }
 
+  if (hasMetadata(todiff, "enzyme_fwdgradient")) {
```

Comment:
Gradient isn't a great name for custom forward pass. Also can we get a test for this?

Reply:
this ended up on the wrong branch.